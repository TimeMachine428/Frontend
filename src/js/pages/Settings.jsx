import React from "react";

export default class Settings extends React.Component {
  login = this.props.location.state.login;

  render() {
    return (
      <h1>Settings</h1>
    );
  }
}
