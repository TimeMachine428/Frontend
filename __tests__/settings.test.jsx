import React from 'react';
import Settings from '../src/js/pages/Settings.jsx';
import renderer from 'react-test-renderer';

test('Settings page information correct',() =>{
    const component = renderer.create(
        <Settings />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})

