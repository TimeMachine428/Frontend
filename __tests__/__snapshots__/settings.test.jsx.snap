// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page information correct 1`] = `
<h1>
  Settings
</h1>
`;
